bgpd/bgp_route.c

```diff
@@ -8617,8 +8617,14 @@ static int bgp_show_route(struct vty *vty, struct bgp *bgp, const char *ip_str,
 			  int prefix_check, enum bgp_path_type pathtype,
 			  u_char use_json)
 {
-	if (!bgp)
+	if (!bgp) {
```

Comment:
Blank returns for json when there is nothing to return is considered invalid json.  We need to return {}